dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/helpers/calculate-initial-utilisation-and-fixed-fee.ts

```diff
@@ -70,12 +70,10 @@ export const calculateInitialUtilisationAndFixedFee = async (facilityId: string)
     throw new Error(`TFM facility values for ${facilityId} are missing`);
   }
 
-  const utilisation = calculateInitialUtilisation(value);
-
-  const ukefShareOfUtilisation = calculateUkefShareOfUtilisation(utilisation, coverPercentage);
+  const ukefShareOfInitialUtilisation = new Big(calculateDrawnAmount(value, coverPercentage)).round(2).toNumber();
```

Comment:
`calculateDrawnAmount` doesn't use Big or any rounding but we need to match what is in ACBS so we have to use the same function and then do the rounding ourselves after here

Reply:
Roughly, we want to change to this because using the same function as what we send to ACBS means we will always be in sync rather than relying on the methods lining up. 

The actual real change and fix is actually further down in what we return.